package.json

```diff
@@ -138,6 +138,12 @@
                     "scope": "resource",
                     "type": "boolean"
                 },
+                "mypy-type-checker.daemonStatusFile": {
+                    "default": "",
+                    "markdownDescription": "%settings.daemonStatusFile.description%",
+                    "scope": "resource",
+                    "type": "string"
+                },
```

Comment:
You can add `"tags": ["experimental"]` to mark a setting a experimental in the UI
						]

Reply:
Added https://github.com/microsoft/vscode-mypy/pull/347/commits/a33124cbcfd19a48ddab06972bc7051d50a5d15f